Libraries/Text/RCTTextAttributes.m

```diff
@@ -183,6 +185,12 @@ - (NSParagraphStyle *)effectiveParagraphStyle
     attributes[RCTTextAttributesIsHighlightedAttributeName] = @YES;
   }
 
+  // [TODO(macOS ISS#2323203)
+  if (_fontSmoothing != RCTFontSmoothingAuto) {
```

Comment:
Is it possible it could get set to something other than auto and then set back to auto (which would then fail to update here)?

Reply:
Hmm, that's a good point.  Although in practice I doubt anyone is going to be toggling this style on and off.   I'll test some stuff out setting it without the condition